docs/tutorials/cwp_calib_tutorial.rst

```diff
@@ -0,0 +1,185 @@
+===========================================================
+Borehole Calibration with Selective Simulation Cancellation
+===========================================================
+
+Introduction - Calibration with libEnsemble and CWP
+---------------------------------------------------
+
+This tutorial demonstrates libEnsemble's capability to selectively cancel pending
+simulations based on instructions from the *Persistent CWP* calibration
+generator function. This capability is critical for this calibration use-case since
+it isn't useful for the generator to receive extraneous evaluations
+from resources that may be more effectively applied towards critical evaluations.
+
```

Comment:
I wouldn't say "critical", but improves efficiency. Maybe simplfy the sentence a bit.

e.g. This improves the efficiency by preventing time spent running obviated simulations.

Reply:
Thank you, I mention the irregular comms patterns in the end of the Generator section, but I could add an introductory remark near the beginning, and expand more later on.

Reply:
Maybe just one sentence to mention uses irregular comms here